src/python/grapl-common/grapl_common/utils/find_grapl_root.py

```diff
@@ -0,0 +1,41 @@
+import subprocess
+from pathlib import Path
+from typing import Optional
+
+
+def find_grapl_root() -> Optional[Path]:
+    # We could potentially add other heuristics here.
+    return _find_grapl_root_based_off_git_in_pwd()
+
+
+def _find_grapl_root_based_off_git_in_pwd() -> Optional[Path]:
+    cmd = "git rev-parse --show-toplevel"
+    git_repo_root = _quietly_execute(cmd)
+    if git_repo_root is None:
+        return None
+
+    git_repo_root_path = Path(git_repo_root)
+    del git_repo_root
```

Comment:
(just removes this variable from being able to be consumed; the path is a better API)